modules/postgres/postgres.go

```diff
@@ -170,6 +170,81 @@ func WithSnapshotName(name string) SnapshotOption {
 	}
 }
 
+// WithSSLSettings configures the Postgres server to run with the provided CA Chain
+// This will not function if the corresponding postgres conf is not correctly configured.
+// Namely the paths below must match what is set in the conf file
+func WithSSLSettings(sslSettings SSLSettings) testcontainers.CustomizeRequestOption {
```

Comment:
Do we want users to customise the certs, or just use the tlscert library to happen under the hood?

Reply:
I like things to work out of the box, so I could see removing that. But inevitably there is going to be a issue opened titled "Allow users to use their own certificate chain"